pysmt/test/test_rewritings.py

```diff
@@ -83,6 +83,67 @@ def test_prenex_examples(self):
                         ok = not logic.quantifier_free
                     self.assertTrue(ok)
 
+    @skipIfNoSolverForLogic(QF_AUFLIA)
+    def test_ackermannization_unary(self):
+        a,b = (Symbol(x, INT) for x in "ab")
+        f,g,h = (Symbol(x, FunctionType(INT, [INT])) for x in "fgh")
+        #f(g(h(a))) = f(g(h(b)))
+        formula1 = Not(Equals(
+                    Function(f,
+                             [Function(g,
+                                       [Function(h, [a])])
+                             ]),
+                    Function(f,
+                             [Function(g,
+                                       [Function(h, [b])])
+                             ]),
+
+                ))
+        formula2 = Equals(a, b)
+        formula = And(formula1, formula2)
+        ack = ackermannize(formula)
+        #verify that there are no functions in ack
+        atoms = ack.get_atoms()
+        for atom in atoms:
+            for arg in atom.args():
+                self.assertFalse(arg.is_function_application())
+        #verify that ack and formula are equisat
+        formula_sat = is_sat(formula, logic=QF_AUFLIA)
+        ack_sat = is_sat(ack, logic=QF_LIA)
+        self.assertTrue(formula_sat == ack_sat)
+
+    @skipIfNoSolverForLogic(QF_AUFLIA)
+    def test_ackermannization_binary(self):
```

Comment:
This test is very similar to `test_ackermannization_unary()`.
What would be the best way of sharing their code? 
I thought about using `get_example_formulae()`, but I do want the specific formulas I created here, not the formulas that are generated there.
And when adding a helper function, it got run as a test, which is undesirable.